From 22c7491891f20909a73ecdd10aadce4b5b8e3da8 Mon Sep 17 00:00:00 2001
From: Kai Bergmann <44040989+kai-li-wop@users.noreply.github.com>
Date: Thu, 15 Jan 2026 16:44:17 +0100
Subject: [PATCH] CI: remove march=k8 reference

There is no reason to explicitly specify the architecture here.
---
 code/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/code/CMakeLists.txt b/code/CMakeLists.txt
index 90f19a8c..5f01cb43 100644
--- a/code/CMakeLists.txt
+++ b/code/CMakeLists.txt
@@ -351,8 +351,6 @@ function(add_asm TARGET)
 	elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "arm" OR "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "arm64")
 	else()
 		target_sources(${TARGET} PRIVATE ${CODE_DIR}/asm/snapvector.c ${CODE_DIR}/asm/ftola.c)
-		set_source_files_properties(${CODE_DIR}/asm/snapvector.c PROPERTIES COMPILE_FLAGS -march=k8)
-		set_source_files_properties(${CODE_DIR}/asm/ftola.c PROPERTIES COMPILE_FLAGS -march=k8)
 	endif()
 endfunction()
 
